docs/api/main-config-typescript.md

```diff
@@ -90,6 +90,8 @@ Only available for React Storybook projects. Options to pass to react-docgen-typ
 
 ## `skipBabel`
 
+Deprecated: Will be removed in Storybook 8.0. Use `skipCompiler` instead
+
 Type: `boolean`
 
 Disable parsing of TypeScript files through babel.
```

Comment:
@valentinpalkovic, when you're able, can you capitalize Babel as we're referring to the tooling and also adjust "Deprecated: Will be removed in Storybook 8.0. Use `skipCompiler` instead"  to "Deprecated: Will be removed in Storybook 8.0. Use `skipCompiler` instead."  as this is a full sentence.